### PR TITLE
Add option for fast open in RocksDb

### PR DESCRIPTION
**Pull request by jbayardo**

### CODE REVIEW

File: Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs [hunk 1]
```diff
@@ -194,9 +194,15 @@ private BoolResult Load(OperationContext context, StoreSlot activeSlot, bool cle
 
                 Tracer.Info(context, $"Creating rocksdb store at '{storeLocation}'.");
 
-                var possibleStore = KeyValueStoreAccessor.Open(storeLocation,
-                    additionalColumns: new[] { nameof(Columns.ClusterState), nameof(Columns.Metadata) },
-                    rotateLogs: true,
+                var possibleStore = KeyValueStoreAccessor.Open(
```
Comment (dcwuser) [inline Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs]: Comments please on what each of these does.

Comment (jbayardo) [reply Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs]: Added inline comments. RocksDb options are commented in the documentation for the arguments object.

---
In reply to: [349360402](https://github.com/microsoft/BuildXL/pull/1257#discussion_r349360402) [](ancestors = 349360402)

**Pull request merged**